macros/conversations.cfg

```
@@ -1,7 +1,7 @@
 #textdomain wesnoth-A_New_Order
 #define AFTERBARNONSTORY
     [part]
-        # Konrad2 suggested replacing "was ended" with "was over" but I think "had ended" works better:
+        #po: Konrad2 suggested replacing "was ended" with "was over" but I think "had ended" works better:
         story=_"When the battle had ended, many bodies were found on fields near Barnon; but Gawen's body was not among them. Furious, Bor Cryne ordered a search of the whole battlefield, but the efforts of his men proved fruitless."
```

I happened to notice that semicolon is grammatically incorrect.

```suggestion
        story=_"When the battle had ended, many bodies were found on fields near Barnon, but Gawen's body was not among them. Furious, Bor Cryne ordered a search of the whole battlefield, but the efforts of his men proved fruitless."
```